src/assets/styles/base.scss

```diff
@@ -142,6 +142,10 @@ p {
   font-weight: normal;
 }
 
+.tool-tip {
+  cursor: default
```

Comment:
Did UX ask for all tooltips to be an arrow instead of a pointing hand?

May I recommend the "help" cursor for tooltips?

Reply:
this is what was requested by UX:

>don't change the mouse cursor to the text tool on hover for tool tips (keep as regular pointer)

Reply:
OK.

Can you ask them if the "help" cursor would be better?

Reply:
sure, will ask

Reply:
reply from Tracey:

> Tracey Lebedovich @tracey.lebedovich
I think just keep it simple and don’t change the cursor.
aka - keep it as a regular pointer cursor
(not text cursor, not help icon)

Reply:
ok